plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties

```diff
@@ -42,7 +42,8 @@ tooltip.staging.bucket=The Google Cloud Storage bucket used to stage files for d
  If not specified, the application''s default code bucket is used.
 
 #Project selector
-projectselector.createproject=Select a GCP Project. Use the Cloud Console to <a href="{0}">create a new project</a>.
+projectselector.selectProject=Select a Google Cloud Platform project
+projectselector.createproject=To create a project, open the <a href="{0}">Cloud Console</a> then refresh this list.
```

Comment:
Needs punctuation. I think  a period after the link is probably best:

To create a project, open the <a href="{0}">Cloud Console</a>. Then refresh this list.

Hmm, actually merely opening the console is not enough. How about:

Create a project in the <a href="{0}">Cloud Console</a>. Then refresh this list.